tests/unit/test_tags_parser.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from pytest_splunk_addon.standard_lib.addon_parser.tags_parser import (
+    TagsParser,
+    unquote,
+)
+
+
+output_to_build = {
+    "eventtype=fiction_for_tags_positive": {
+        "tags_positive_event": "enabled",
+        "tags_disabled_event": "disabled",
+    },
+    "source=%2Fopt%2Fsplunk%2Fvar%2Flog%2Fsplunk%2Fsplunkd.log": {
+        "tags_positive_event": "enabled",
+        "tags_disabled_event": "disabled",
+    },
+}
+
+
+def test_tags_can_be_parsed_and_extracted(parser_instance):
+    assert hasattr(
+        parser_instance.tags, "sects"
+    ), "tags can not be called or does not have sects attribute"
+
+
+def test_tags_can_be_parsed_and_returned(parsed_output, parser_instance):
+    expected_outputs = []
+    for stanza, section in parsed_output.items():
+        stanza = stanza.replace("=", '="')
+        stanza = unquote('{}"'.format(stanza))
+        for item, value in section.options.items():
+            expected_outputs.append(
+                {
+                    "stanza": stanza,
+                    "tag": value.name,
+                    "enabled": True if value.value == "enabled" else False,
+                }
+            )
```

Comment:
It mirrors logic from source code. I think it is better to use raw data samples in assertions. We should avoid code duplications in tests